### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by yirano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
+      ```
+      yarn dev
+      ```
+    - Access your application at `http://localhost:3000`
+
+## Objectives
+  Clone the respository and complete the following tasks.
+
+### User Story
+  Customer is shown a responsive marketplace where they can add products to the cart. For this test, the customer will need to add more than one product to the cart before proceeding to checkout (figure out how this is going to work). In checkout, customers will be prompted for a delivery address on the first product that they can use for the second product. Each product in the cart will have a service fee that the customer can see as part of the overall total.
```
Comment (ianbautista) [inline README.md]: Maybe rephrase it to "...In checkout, customers will be prompted for a delivery address. Inputted addresses should become delivery options for the other products." and "Each vendor charges a service fee that the customer can see as part of the overall total."

File: src/pages/index.tsx [hunk 2]
```diff
@@ -15,16 +14,7 @@ const Home: React.FC = () => {
...
-      const { data: productsRes } = await axios.get<Product[]>(
-        `${process.env.NEXT_PUBLIC_API_URL}/products`
-      )
-      if (productsRes) {
-        setProducts(productsRes)
-      }
-    } catch (error) {
-      console.error(error)
-    }
+    // endpoint: `${process.env.NEXT_PUBLIC_API_URL}/products`
```
Comment (ianbautista) [inline src/pages/index.tsx]: is this line intentionally commented out?

**Review by ianbautista — CHANGES_REQUESTED** [review]
Love this. Just a few comments.

File: src/pages/index.tsx (hunk 2 above)
Comment (yirano) [reply src/pages/index.tsx]: Yes! I wanted them to complete it but as discussed just earlier I will update this comment

**Review by ianbautista — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,50 +1,49 @@
...
+### Functional Requirements
+  - **Marketplace:**
+    - Utilize the provided endpoint to fetch marketplace data and render a responsive interface.
+      - It's important to note that we're also interested in observing your debugging process.
+    - Allow customers to add products to their cart. A minimum of two products is required to proceed to checkout.
+  - **Error Management:**
+    - Manage errors effectively (and proactively) for both the user and developer.
+    - **BONUS:** Develop an error logger to log messages directly to the console.
+  - **Service Fee Integration:**
+    - Create a `/service-fee` endpoint that returns the service fee set for particular vendors and add that to the total of the cart.
```
Comment (ianbautista) [inline README.md]: I would just rephrase this to "Create a fees endpoint that returns the service fee set for particular vendors and add that to the total of the cart."

Comment (ianbautista) [reply README.md]: So that its not deceiving. Reading the BE Readme will instruct them what that endpoint needs to be.

Comment (yirano) [reply README.md]: @ianbautista Done!

**Review by ianbautista — APPROVED** [review]
